Codeforces_problems/Reverse Binary Strings/problem.md

```diff
@@ -0,0 +1,3 @@
+# Reverse Binary Strings
+
+problem Link: https://codeforces.com/blog/entry/84149
```

Comment:
The problem link seems to be incorrect.
Could you please recheck

Reply:
It is resolved now! Sorry, must have copied the link.